src/shared/proto/tbots_software_msgs.proto

```diff
@@ -0,0 +1,25 @@
+syntax = "proto3";
+
+import "shared/proto/vision.proto";
+import "shared/proto/radio_primitive.proto";
+
+message VisionMsg
+{
+    // epoch timestamp of captured vision data
+    uint64 timestamp = 1;
+
+    // Robot ID to RobotState map
+    map<uint32, RobotState> robot_states = 2;
+
+    // ball state
```

Comment:
Nit: Most of the comments begin capitalized except for those beginning with 'epoch' and 'ball' and a few others earlier. Might be best to keep it consistent just to make it cleaner.

Reply:
fixed